test/language/module-code/instn-star-as-props-dflt-skip.js

```diff
@@ -40,18 +40,19 @@ info: |
   c. For each element n of starNames, do
   i. If SameValue(n, "default") is false, then
   [...]
-flags: [module, export-star-as-namespace-from-module]
+flags: [module]
+features: [export-star-as-namespace-from-module]
   ---*/
 
-import named from './instn-star-props-dflt-skip-star-as-named_FIXTURE.js';
-import production from './instn-star-props-dflt-skip-star-as-prod_FIXTURE.js';
+import {named} from './instn-star-props-dflt-skip-star-as-named_FIXTURE.js';
+import {production} from './instn-star-props-dflt-skip-star-as-prod_FIXTURE.js';
 
 assert('namedOther' in named);
 assert.sameValue(
-  'default' in named, false, 'default specified via identifier'
+  'default' in named, true, 'default specified via identifier'
 );
```

Comment:
☝️ Where is `default` coming from. The `'./instn-star-props-dflt-skip-star-as-named_FIXTURE.js'` exports `named` which is an `export * as named from "itself"`.